app/src/main/java/org/sil/gatherwords/SessionActivity.java

```diff
@@ -18,35 +19,43 @@
 import android.widget.ArrayAdapter;
 import android.widget.EditText;
 import android.widget.Spinner;
+import android.widget.TextView;
 
 import com.google.android.gms.location.FusedLocationProviderClient;
 import com.google.android.gms.location.LocationServices;
 
 import org.sil.gatherwords.room.AppDatabase;
 import org.sil.gatherwords.room.Session;
+import org.sil.gatherwords.room.SessionDao;
 
+import java.lang.ref.WeakReference;
 import java.text.SimpleDateFormat;
 import java.util.Date;
+import java.util.List;
 import java.util.concurrent.ExecutionException;
 
 public class SessionActivity extends AppCompatActivity implements AdapterView.OnItemSelectedListener {
+    EditText dateField, timeField, timeZoneField, labelField, speakerField, eliciterField;
+    SimpleDateFormat dateSDF, timeSDF, timeZoneSDF;
     // Used to track location through multiple methods
     private FusedLocationProviderClient mFusedLocationClient;
     boolean locationEnabled;
     Location location;
     boolean creatingNewSession;
-    AppDatabase db;
+    int id;
```

Comment:
As a member variable, this is a very generic name.  Simply seeing it gives little context of what it is an id for.